decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -52,15 +54,35 @@ module Admin
             subject.map_model(post)
           end
 
-          it "does not assign the user group for normal users" do
-            expect(subject.user_group_id).to be_nil
+          it "assigns current_user as user_group_id" do
+            expect(subject.user_group_id).to eq("current_user")
```

Comment:
```suggestion
            expect(subject.user_group_id).to eq("current_organization")
```
In the example I've put before, the default status of the `author_id` would be "current_organization", so that's the result we are expecting here. You can also decide to start testing with the "current_user" status. Up to you!